docs/README_sp.md

```diff
@@ -0,0 +1,195 @@
+![npm-audit-periodic](https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/workflows/npm-audit-periodic/badge.svg)
+
+<p align="center">
+  <span>Español</span> |
+  <a href="https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/blob/master/README.md">English</a> |
+</p>
+
+# Aplicación Web de referencia para Cloud Run for Anthos
+
+Este repositorio, que incluye todos los flujos de trabajo y automatizaciones asociados, representa un conjunto de mejores prácticas destinadas a demostrar una arquitectura de referencia para crear una aplicación web en Google Cloud utilizando Cloud Run para Anthos.
```

Comment:
What do you think of using _dirigidas_ instead of _destinadas_?

Reply:
Sounds good to me